### PR TITLE
Elizabeth/4 card display

### PR DESCRIPTION
**Pull request by elizabeth-zacharia**
#7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@mui/icons-material": "^5.15.20",
+    "@mui/lab": "^5.0.0-alpha.170",
```
Comment (mzachari) [inline package.json:9]: where are we using this package?

File: src/components/addNote/index.tsx [hunk 2]
```diff
@@ -2,10 +2,26 @@ import { Button } from '@mui/material';
 import Box from '@mui/material/Box';
```
Comment (mzachari) [inline src/components/addNote/index.tsx:2]: to be consistent with the the other folder names, can you name this `AddNote`

**Review by mzachari — CHANGES_REQUESTED** [review]
good work overall, gave some minor comments

File: package.json (hunk 1 above)
Comment (elizabeth-zacharia) [reply package.json:9]: for masonry

File: src/components/addNote/index.tsx (hunk 2 above)
Comment (elizabeth-zacharia) [reply src/components/addNote/index.tsx:2]: yes

**Review by mzachari — APPROVED** [review]

**Pull request merged**